### PR TITLE
Jc-2

### PR DESCRIPTION
**Pull request by freesky15**

### CODE REVIEW

**Comment by saffchen** [comment]
Please change target branch

File: Dockerfiles/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM openjdk:11
```
Comment (pelmegov) [inline Dockerfiles/Dockerfile:1]: Why not 17th?

File: src/main/java/org/jrmjacp/processingservice/utils/DockerProcessing.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import com.github.dockerjava.transport.DockerHttpClient;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.apache.commons.io.IOUtils;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.time.Duration;
+
+
```
Comment (pelmegov) [inline src/main/java/org/jrmjacp/processingservice/utils/DockerProcessing.java:19]: Setup autoformatting before commit.

**Review by pelmegov — CHANGES_REQUESTED** [review]

**Comment by freesky15** [comment]
Done!

**Pull request closed without merging**